src/webapp/pages/form-page/useForm.ts

```diff
@@ -65,12 +66,22 @@ export function useForm(formType: FormType, id?: Id): State {
     const [configurableForm, setConfigurableForm] = useState<ConfigurableForm>();
     const [formLabels, setFormLabels] = useState<FormLables>();
     const [isLoading, setIsLoading] = useState(false);
+    const [currentEventTrackerState, setCurrentEventTrackerState] =
+        useState<Maybe<DiseaseOutbreakEvent>>();
     const currentEventTracker = getCurrentEventTracker();
     const { existingEventTrackerTypes } = useExistingEventTrackerTypes();
     useCheckWritePermission(formType);
     const snackbar = useSnackbar();
 
     useEffect(() => {
+        // NOTICE: This if is to avoid infinite loop when the form page is reloaded from browser
+        if (
+            currentEventTrackerState &&
```

Comment:
@anagperal With this change, multi step forms stop working. For example, risk and IAP are multi step forms. Clicking on next does not go to the next page. PFA video.  

https://github.com/user-attachments/assets/76afcd18-bebd-4643-903f-6213b507c44e


https://github.com/user-attachments/assets/83817c59-e603-41bc-b2bf-cfbb9c6fc089

Reply:
> @anagperal With this change, multi step forms stop working. For example, risk and IAP are multi step forms. Clicking on next does not go to the next page. PFA video.
> 
>  risk-multi-step-not-working.mov 
>  iap-multi-step-not-working.mov

Soled! Check if this is correct https://github.com/EyeSeeTea/zebra-dev/pull/36/commits/368eb3e74c3f99bbf3f08fbec181402bdedea63b

Reply:
@anagperal Thank you! this was a miss on my end , definitely need this change.